packages/remix-testing/create-remix-stub.tsx

```diff
@@ -33,27 +38,112 @@ type RemixStubOptions = {
    *   initialIndex: 1 // start at "/events/123"
    */
   initialIndex?: number;
+
+  remixConfigFuture?: Partial<FutureConfig>;
 };
 
-export function createRemixStub(routes: RouteObject[]) {
+type StubRouteObject = Omit<
+  RouteObject,
+  "id" | "handle" | "shouldRevalidate" | "errorElement" | "hasErrorBoundary"
```

Comment:
I could see some of these being useful for more advanced tests/stories.  There's no reason you couldn't force an error to bubble to an `errorElement` on your route, or access something via `useMatches` with `id`/`handle`, or opt out of a `fetcher.load` revalidation etc.  I think if we `Omit`, we'll just get folks asking us to un-omit eventually (or they'll just `@ts-expect-error` and do it anyway 😂 ).  

`hasErrorBoundary` could probably be omitted (but I don't think it needs to be) since it'll get added in `createMemoryRouter`